.spelling

```diff
@@ -138,8 +140,11 @@ wildcard
 wildcards
 WIP
 x509
+yaml
 YAML
 YAMLs
+v0.15
```

Comment:
So we had some discussion about this before, we should always place version in backticks in the docs so we don't end up with all versions here